torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3349,8 +3391,6 @@ def method_tests():
         ('index_copy', (S, S), (0, index_perm_variable(2, S), (2, S)), 'dim', (), [0]),
```

Comment:
Can these other index_copy entries in method_tests() be removed to? Do new sample inputs need to be created for them?

Reply:
The first one I have removed. The second one is not so clear how to implement it with  `OpInfo`, as it has an `expectedAlertNondeterministic('index_copy')`. How can that be translated to an `OpInfo` test?